src/open_inwoner/components/templates/components/Card/CardContainer.html

```diff
@@ -4,29 +4,29 @@
         {% for category in categories %}
             {% url 'products:category_detail' slug=category.slug as category_url %}
             {% if category.icon %}
-                {% card src=category.icon.file.url alt=category.icon.name title=category.name href=category_url image_object_fit=image_object_fit %}
+                {% card src=category.icon.file.url alt=category.icon.name title=category.name href=category_url compact=True image_object_fit=image_object_fit %}
```

Comment:
I don't mind these changes, but is this related to issue [1982](https://taiga.maykinmedia.nl/project/open-inwoner/task/1982)?

Reply:
It actually is, yes - The 'drag' field here was no longer conforming to the design because things changed in the Card paddings in another PR: some cards now have different vertical padding + different horizontal padding, but other Cards still have the same padding on all sides.